src/Rendering/ComponentParameter`1.cs

```diff
@@ -28,67 +28,67 @@ namespace Bunit
         /// </summary>
         public bool IsCascadingValue { get; }
 
-        private ComponentParameter(Expression<Func<TComponent, TValue>> expression, [AllowNull] TValue value, bool isCascadingValue)
+        private ComponentParameter(Expression<Func<TComponent, TValue>> parameterSelector, [AllowNull] TValue value, bool isCascadingValue)
         {
-            if (expression == null)
+            if (parameterSelector == null)
             {
-                throw new ArgumentNullException(nameof(expression));
+                throw new ArgumentNullException(nameof(parameterSelector));
             }
 
             if (isCascadingValue && value is null)
             {
                 throw new ArgumentNullException(nameof(value), "Cascading values cannot be set to null");
             }
 
-            Name = GetParameterNameFromMethodExpression(expression);
+            Name = GetParameterNameFromParameterSelectorExpression(parameterSelector);
             Value = value;
             IsCascadingValue = isCascadingValue;
         }
 
-        private static string GetParameterNameFromMethodExpression<T>(Expression<Func<TComponent, T>> expression)
+        private static string GetParameterNameFromParameterSelectorExpression<T>(Expression<Func<TComponent, T>> parameterSelector)
```

Comment:
This feels needlessly long... maybe:
```suggestion
        private static string GetParameterNameFromExpression<T>(Expression<Func<TComponent, T>> parameterSelector)
```